src/Microsoft.Health.Fhir.SqlServer/Features/Storage/SqlServerFhirModel.cs

```diff
@@ -189,7 +195,7 @@ COMMIT TRANSACTION
                     sqlCommand.Parameters.AddWithValue("@claimTypes", commaSeparatedClaimTypes);
                     sqlCommand.Parameters.AddWithValue("@compartmentTypes", commaSeparatedCompartmentTypes);
 
-                    using (SqlDataReader reader = await sqlCommand.ExecuteReaderAsync(CommandBehavior.SequentialAccess))
+                    using (SqlDataReader reader = await sqlCommand.ExecuteReaderAsync(CommandBehavior.SequentialAccess).ConfigureAwait(false))
```

Comment:
in .net core, this is not needed

Reply:
Did a bit a of reading about this... is this the reason why?

"_code running in an ASP.NET Core app by default won’t see a custom SynchronizationContext, which lessens the need for ConfigureAwait(false) running in such an environment._" (taken from [here](https://devblogs.microsoft.com/dotnet/configureawait-faq/))

Reply:
Yeah. What I said was technically not true, but we're not publishing these as general-purpose libraries, and we don't call ConfigureAwait(false) elsewhere, because we don't create a synchonization context and neither does asp.net core. 
You could certainly make the case that we should. But then we should do it everywhere and turn on an analyzer rule to enforce it. I'm not sure the added noise would be worth it.

Reply:
Okay, makes sense. Thanks for the background!

Reply:
Removed in 1079d38.